src/test/java/org/jnode/fs/FileSystemTestUtils.java

```diff
@@ -101,4 +84,46 @@ public static byte[] intsToByteArray(String signedInts) {
 
         return bytes;
     }
+
+    /**
+     * Converts a string of signed ints to a byte array.
+     *
+     * @param ints the int array.
+     * @return the byte array.
+     */
+    public static byte[] intArrayToByteArray(int[] ints) {
```

Comment:
I think we should just save the raw data to a file instead of keeping the `int[]` data in `CompressedTestData`

Reply:
The test data is coped from https://github.com/libyal/libfsntfs/blob/82181db7c9f272f98257cf3576243d9ccbbe8823/tests/fsntfs_test_compression.c#L39 directly. I feel it is easier to compare them if I keep the format unchanged. (E.g. if they update the data later, we can easily apply to our test code here.)

Reply:
But the test should never really change because the algorithm is not changing